packages/paste-website/src/pages/components/switch/index.mdx

```diff
@@ -177,39 +210,57 @@ yarn add @twilio-paste/switch - or - yarn add @twilio-paste/core
 #### Usage
 
 ```jsx
-import {Switch} from '@twilio-paste/core/switch';
+import {Switch, SwitchGroup} from '@twilio-paste/core/switch';
 export const Basic = () => {
   return (
-    <SwitchContainer label="label text">
-      <Switch />
-    </SwitchContainer>
+    <SwitchGroup name="bar" legend="This is the legend text" required disabled>
+      <Switch value="1" helpText="This is some help text.">
+        First
+      </Switch>
+      <Switch value="2" helpText="This is some help text.">
+        Second
+      </Switch>
+      <Switch value="3" helpText="This is some help text.">
+        Third
+      </Switch>
+    </SwitchGroup>
   );
 };
 ```
 
-#### SwitchContainer Props
+#### Switch Props
```

Comment:
suggestion: can we fill in some of the missing descriptions and defaults for switch and switchgroup? some of them are obvious but all the blanks looks a bit unfinished